examples/Test/Automated/readme.md

```diff
@@ -0,0 +1,18 @@
+This folder contains the HL7 sample files that are used in the automated ReportStream
+integration tests. The [automated-staging-test-submit.yml](../../../.github/workflows/automated-staging-test-submit.yml)
```

Comment:
If you want, you can make this path absolute instead of relative to the current location of this readme.

```suggestion
integration tests. The [automated-staging-test-submit.yml](/.github/workflows/automated-staging-test-submit.yml)
```